recordings-bulk-actions/config.yml

```diff
@@ -20,6 +20,13 @@ steps:
   - title: Execute Job
     content: |
       The user will specify the job_id and use the PUT for excuting the bulk job
+  - title: Cancel Job
+      content: |
+        The user will specify the job_id and use the DELETE for canceling the bulk job
+  - title: Get Several Jobs
+      content: |
+        The user will specify the parameters in a dictionary as in the example and will get all jobs satisfying the filters
+        and in the requested order.
 
 languages:
   python:
```

Comment:
The first step should also be updated to include all lines in the file (0-67 -> 0-91)

Reply:
done